### PR TITLE
PLANET-6952: New layout applied to default and category listing pages

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-6952

## Description
Based on the Action listing page

Demo page: https://www-dev.greenpeace.org/test-pandora/listing-page/

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [leda](https://www-dev.greenpeace.org/test-leda) | [admin](https://www-dev.greenpeace.org/test-leda/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-leda/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-leda) | [composer-local.json](https://github.com/greenpeace/planet4-test-leda/blob/main/composer-local.json)

:watch: 2022.11.09 20:42:56

File: assets/src/scss/layout/_block-classes.scss [hunk 1]
```diff
@@ -1,94 +1,3 @@
-.wp-block-query-pagination {
```
Comment (GP-Dan-Tovbein) [inline assets/src/scss/layout/_block-classes.scss:1]: Needed to move these styles to the new released listing page's stylesheet.

Comment (mleray) [reply assets/src/scss/layout/_block-classes.scss:1]: Why aren't you also moving the other styles from this CSS file? It seems they are also related to the listing pages 🤔 
Also, [this `speak` property](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/layout/_block-classes.scss#L101) that we use is currently not supported by any browser, so I'm not sure why it's there 😅

Comment (GP-Dan-Tovbein) [reply assets/src/scss/layout/_block-classes.scss:1]: It's been moved to [here](https://github.com/greenpeace/planet4-master-theme/pull/1839/files#diff-215e8c1cad357161f9ea21bc668f3da5a9a489a79d302553823b12e1eaf5acf2R198)!

Comment (GP-Dan-Tovbein) [reply assets/src/scss/layout/_block-classes.scss:1]: > Also, this speak property that we use is currently not supported by any browser, so I'm not sure why it's there 😅

Only [Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style/speak-as#browser_compatibility)

Comment (mleray) [reply assets/src/scss/layout/_block-classes.scss:1]: That's for the `speak-as` property, not `speak`

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**